### ISSUE TITLE
Exception with "Show hidden records" and a FluidContent element from other page

### ISSUE THREAD

**Issue by lars85** [FluidTYPO3/fluidcontent#202]
1. Create a fluid content element on Page A
2. Go to Page B in the Backend
3. Create a "Insert Record" element and select the fluid content element from Page A
4. Go to the frontend page of Page B
5. Enable "show hidden records" in the Admin panel
![screen shot 2014-12-10 at 12 15 21](https://cloud.githubusercontent.com/assets/1326345/5375056/425b04fa-8066-11e4-84fc-8cba44a6b4d4.png)

The frontend page shows the exception

    #1225709595: Template could not be loaded. I tried "/Users/lm/Workspace/sma/lp/htdocs/typo3conf/ext/fluidcontent/Resources/Private/Templates/Content/Default.html" (More information)
    TYPO3\CMS\Fluid\View\Exception\InvalidTemplateResourceException thrown in file
/Users/lm/Workspace/sma/lp/htdocs/typo3_src/typo3/sysext/fluid/Classes/View/TemplateView.php in line 370.

![screen shot 2014-12-10 at 12 21 59](https://cloud.githubusercontent.com/assets/1326345/5375165/45dd85e8-8067-11e4-873e-b14dc3f1f472.png)

I am using this TypoScript to change the directories of the partials and layouts:

    plugin.tx_ownextension {
    	view {
    		templateRootPath = EXT:own_extension/Resources/Private/Templates/
    		partialRootPath = EXT:own_extension/Resources/Private/Partials/
    		layoutRootPath = EXT:own_extension/Resources/Private/Layouts/
    	}
    }

I guess this typoscript is not loaded for this content element on Page B.

When I call page A everything works like expected without a exception.

Versions:
- fluidcontent: 4.1.1 (489c7ebc11d18729561a37a911f9afdaf45269f2)
- flux: dev-staging (9a40fdc21b13a28a43abe00d406b71878623a28f)
- TYPO3 Core: 6.2.6

**Comment by NamelessCoder** [comment]
@lars85 Happy new year! Any updates about this issue? ;)

**Comment by lars85** [comment]
I am not able to reproduce this bug with TYPO3 6.2.9, Flux 7.1.2 and FluidContent 4.1.1..

**Comment by Crewcut** [comment]
Just got excact same error on my installation (TYPO3 6.2.9).
![skaermbillede 2015-01-17 kl 11 17 03](https://cloud.githubusercontent.com/assets/8476954/5788769/a6ee584e-9e49-11e4-9971-7a3204440ae5.png)
Problems startet after updating fluidtypo3 extensions. Tried to remove various extensions but no fix with that. Strange thing is, that i have two different websites on the same installation, one has this problem, the other has not.  I thought it was related to namespaces, but tried all variables of that (new and old). Content elements in BE does not show other than wording 'Fluid Content'.
![skaermbillede 2015-01-17 kl 13 03 54](https://cloud.githubusercontent.com/assets/8476954/5788766/89c3c9fc-9e49-11e4-9028-2687bd8c2a95.png)
When using fluidbootstraptheme alone (my own ext. turned off) gives same error. It does look like no template is being parsed.
Tried to delete the page template in the database field manually, that did not help either.

I really have no idea where to look for this - hope someone can help!

Regards,
Kim Holberg (Crewcut)

**Comment by xf-** [comment]
hi, the path exists? - looks absolute and httpdocs as root folder. 
Test file_get_contents in a single file

Cache cleared in install tool - Db compare (maybe some other change)

**Comment by Crewcut** [comment]
I cleared cache in install tool, compared DB - no problem. The other suggestions i do not know how to excecute. Which path are you referring to? How can i test file_get_contents in a single file?

Sorry i am not that experienced with debugging

Kim

**Comment by xf-** [comment]
```php
<?php
file_get_contents('your_file_path');
```
Ensure error_reporting is on and display errors (or take a look in php log file).

**Comment by Crewcut** [comment]
Found out what was causing the problem here. I recently updated all fluidTYPO3 extensions and replaced fluidcontent_bootstrap with fluidbootstraptheme, since the fluidcontent_boostrap was depreaciated. I assumed, that i could replace it without problems, which i could not. Installing the old extension got the website back on.

Question here is, what is best practice to replace fluidcontent_bootstrap with fluidbootstraptheme?
